### DISCUSSION TITLE
Is it possible to get the character in a specific cell?

### DISCUSSION THREAD

**Discussion by ValerioB88** [sourcelair/xterm.js#5040] [Q&A] [answered]
Let's say I want to know the content of the cell at a certain row and col. How woul I do that? Is it even possible?

**Comment by jerch** [comment, marked as answer]
Yes it is possible, e.g. `term.buffer.active.getLine(0).getCell(0).getChars()` would give you the characters at cell [0, 0] on the active buffer.

Also see https://github.com/xtermjs/xterm.js/blob/1709d4469762033ad4e0e56098e32379068fc3b4/typings/xterm.d.ts#L1463-L1699

**Comment by ValerioB88** [comment]
thanks!
